### ISSUE TITLE
Earlham unable to attach works to collections

### ISSUE THREAD

**Issue by jillpe** [scientist-softserv/palni-palci#336]
# Summary
[Earlham](http://earlham.hykucommons.org/) unable to attach works to collections

# Accepted Criteria
- [ ] Earlham is able to attach works to collections

**Comment by labradford** [comment]
https://sentry.io/organizations/scientist-inc/issues/3781022422/?project=6707374&query=is%3Aunresolved&referrer=issue-stream

**Comment by labradford** [comment]
https://assaydepot.slack.com/archives/C0313NKC08L/p1669925598052069

**Comment by bkiahstroud** [comment]
What I was able to replicate: 

1. Error when attaching works to collections :x: 
2. Timeout when attaching works to collections :x: 
3. Collections not showing up in search bar when trying to add works ✅ 
4. Message saying I don't have access to any existing collections :x: 

---

3) For the `Mount Pleasant Free Labor Store Collection` (ID: `db416e6b-d472-40f9-ab52-94038e5ac451`) collection in the Earlham tenant, I was able to narrow this down to the admin group's "manage" access missing from the Sharing tab:

<details><summary>Screenshot</summary>

![Edit User Collection Mount Pleasant Free Labor Store Collection Earlham Friends Collection and College Archives 2022-12-20 at 3 54 19 PM](https://user-images.githubusercontent.com/32469930/208789197-c305fc1c-a03b-45a2-a607-ed1f6efe8501.jpg)

</details> 

Granting the admin group manage access led to the collection showing up in the search results again (for admins). 

I did not run into this same issue for any other collections in the Earlham tenant. 

---

As for the other 3 reported issues, I wasn't able to replicate any of these. It's possible these could have been caused by instability on the server that has since been resolved. 

**TL;DR** I haven't been able to reproduce most of the reported issues. I'm pushing this forward for testing to uncover whether users are still running into these problems
